api/ProfileName/[profileAddress].ts

```diff
@@ -0,0 +1,59 @@
+import assert from 'assert';
+
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+import { isAddress } from 'ethers/lib/utils';
+
+import { adminClient } from '../../api-lib/gql/adminClient';
+import { errorResponse, NotFoundError } from '../../api-lib/HttpError';
+
+const CACHE_SECONDS = 60 * 5;
+export default async function handler(req: VercelRequest, res: VercelResponse) {
+  try {
+    let address: string | undefined;
+    if (typeof req.query.profileAddress == 'string') {
+      address = req.query.profileAddress;
+    } else if (Array.isArray(req.query.address)) {
+      address = req.query.profileAddress.pop();
+    }
+
+    if (!address || !isAddress(address)) {
+      throw new NotFoundError('no valid address provided');
+    }
+
+    address = address.toLowerCase();
```

Comment:
we're doing an ilike in the query, so this isn't actually necessary, right? doesn't hurt to do I guess...